build/BranchInfo.props

```diff
@@ -2,8 +2,8 @@
   <PropertyGroup>
     <MajorVersion>0</MajorVersion>
     <MinorVersion>102</MinorVersion>
-    <PatchVersion>7</PatchVersion>
-    <PreviousPackageVersion>0.102.6</PreviousPackageVersion>
+    <PatchVersion>8</PatchVersion>
```

Comment:
Actually, 0.102.7 hasn't been released, so there's no need to bump. I usually bump the version number right after a release of the previous version, helps me not forget to do it.